test/powershell/engine/WildcardPattern.Tests.ps1

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe "WildcardPattern.ToRegex Tests" -Tags "CI" {
+    Context "Basic wildcard to regex conversion" {
```

Comment:
It seems most of new tests could be in TestCases excluding `Context "Edge cases"`